app/views/comments/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<p><%= parent_comment.text %></p>
+<%= form_for(comment, url: post_comments_path(parent_post.id), method: :post) do |f| %>
+  <%= f.text_area :text %>
+  <br />
+  <%= f.hidden_field :post_id, :value => parent_post.id %>
+  <%= f.hidden_field :comment_id, :value => parent_comment.id %>
+  <% flash.each do |type, msg| %>
+    <% if type.to_s == "inline" %>
+      <%= content_tag(:div, msg, class: "alert alert-alert alert-inline") %>
+    <% end %>
+  <% end %>
+  <%= f.submit "Submit Reply" %>
+<% end %>
```

Comment:
is this the same form that shows up on `posts#show`? If so, can you think of a way to share it?

Reply:
I've just made a list of aspects that make each form distinct from one another. 

In form_for options hash:
  new: `url: post_comments_path(parent_post.id)`
  show: `url: post_comments_path(post.id)`

Within form body:
  new: `<%= f.hidden_field :comment_id, :value => parent_comment.id %>`
  show: ^^ Does not have the line above

  new: ` <%= f.hidden_field :post_id, :value => parent_post.id %>`
  show: ` <%= f.hidden_field :post_id, :value => post.id %>`

Given that all of these are related to naming a parent, it seems like this could be a case where making Comments polymorphic as commentable could offer an advantage. From the Rails guide, "If you have an instance of the Picture model, you can get to its parent via @picture.imageable." 
[http://guides.rubyonrails.org/association_basics.html#polymorphic-associations](url)

Reply:
https://github.com/gingin77/hackernews_clone/issues/18

Reply:
that's an awesome list -- well done!